USERS.md

```diff
@@ -0,0 +1,14 @@
+# Users Guide
```

Comment:
Non-blocking: are we going to further explain what kind of users are the target of this project in future work?

Reply:
Yes. Maybe in the future?

Reply:
Non-blocking means I approve and all comments are at your discretion for now or later. I am resolving this one.